apps/frontend/core-dapp/src/lib/core-contracts.ts

```diff
@@ -58,6 +59,12 @@ export const DEPOSIT_RECORD_ADDRESS: ExternalContract = {
   goerli: '0x1412039d105c41d19f8c77aebdc118a5945aba6a',
 }
 
+export const TOKEN_SENDER_ADDRESS: ExternalContract = {
+  // A dummy contract for testing purposes only.
```

Comment:
Let's add a TODO here
```suggestion
  // TODO: update to actual contract address
  // A dummy contract for testing purposes only.
```